client/src/App.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect,
+} from 'react-router-dom';
+
+import * as Routes from './routes';
+
+import Home from './pages/Home';
+import Member from './pages/Member';
+import Login from './pages/Login';
+
+import Navbar from './components/navbar/Navbar';
+
+import {
+  endUserSession,
+  getUserAuth,
+  startUserSession,
+} from './utils/apiWrapper';
+
+function App() {
+  const [isAuthenticated, setIsAuthenticated] = React.useState(false);
+
+  React.useEffect(() => {
+    const userAuth = async () => {
+      const resp = await getUserAuth();
+      if (!resp.error) setIsAuthenticated(resp.data.result);
+    };
+    userAuth();
+  }, []);
+
+  const login = () => {
+    // call backend api/auth/login
+    const startSession = async () => {
+      const resp = await startUserSession();
+      if (!resp.error) setIsAuthenticated(true);
+    };
+    setIsAuthenticated(true);
```

Comment:
I haven't actually established the backend logic for login authentication because I am waiting for the backend to merge to actually pull the data properly. The two setIsAuth... you see here are providing the logic as follows:
1. Make API call to backend to start user session and if it doesn't throw an error of any sorts Authenticate the user (the backend will save the status in the /auth endpoint and also a cookie from what I understand. 
2. The second setIsAuth will not exist when the API is connected properly, right now, this line of logic exists because there is no connection to the backend to validate a users session so it would never actually login a user. This setIsAuth automatically validates the user as authenticated without the need of the back end for now. The default state of setIsAuth is already false, so I won't need to reset it to false in this method, just determine when it gets set to true